1999Home.html

```diff
@@ -54,9 +54,9 @@
     <div class=webtv-today-container>
       <div class=today-content-container>
         <div class=today-header></div>
-        <div class=today-main>Welcome to WebTV HD!
+        <div class=today-main>WebTV HD Fork recieving a lot of development!
```

Comment:
"A lot of development", huh? All you did was change text and slap your name everywhere you could.